core_debug.tex

```diff
@@ -66,6 +66,12 @@ \section{Load-Reserved/Store-Conditional Instructions}
     automate this.
 \end{commentary}
 
+\section{Wait for Interrupt Instruction}
+
+If halt is requested while {\tt wfi} is executing, then the hart must leave the
+stalled state, completing this instruction's execution, and then enter Debug
```

Comment:
is there a reason to use "stalled' here and "wait" in your other change? Aren't they the same state? Can we use the same term?

Reply:
I'll make it consistently use stalled (which is the term used in the priv spec).